ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -611,8 +611,13 @@ - (void)webViewWebContentProcessDidTerminate:(WKWebView *)webView
 
 - (void)webView:(WKWebView *)webView decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler {
   if (_onNavigationResponse) {
-    NSDictionary *headers = ((NSHTTPURLResponse *)navigationResponse.response).allHeaderFields;
-    NSInteger statusCode = ((NSHTTPURLResponse *)navigationResponse.response).statusCode;
+    NSDictionary *headers = @{};
+    NSInteger statusCode = 200;
+    if([navigationResponse.response isKindOfClass:[NSHTTPURLResponse class]]){
+        headers = ((NSHTTPURLResponse *)navigationResponse.response).allHeaderFields;
+        statusCode = ((NSHTTPURLResponse *)navigationResponse.response).statusCode;
+    }
```

Comment:
How is this part related to the symbol clash and rename?

Reply:
RN 0.57 duplicate class `RCTWebView` ` RCTWebViewManager`. Error will occur when Xcode build.
![image](https://user-images.githubusercontent.com/8196592/46239954-d4784380-c3d2-11e8-800b-764e65583bd6.png)

Reply:
I know about the duplicate symbol error. But the specific lines of code I commented on do not relate to those symbols. It looks like a different commit accidentally came along for the ride.

Reply:
I think this happens to be a fix for #199

Reply:
I think this happens to be a fix for #199